sparkplug_b/sparkplug_b_c_sharp.proto

```diff
@@ -201,10 +197,10 @@ message Payload {
         }
     }
 
-    uint64   timestamp     = 1;        // Timestamp at message sending time
-    repeated Metric metric = 2;        // Repeated forever - no limit in Google Protobufs
-    uint64   seq           = 3;        // Sequence number
-    string   uuid          = 4;        // UUID to track message type in terms of schema definitions
-    bytes    body          = 5;        // To optionally bypass the whole definition above
+    uint64   timestamp      = 1;        // Timestamp at message sending time
+    repeated Metric metrics = 2;        // Repeated forever - no limit in Google Protobufs
```

Comment:
This repeated property is supposed to be named metrics with an s on the end, but was incorrectly named metric and resulted in the c# client library being incorrect.